### PR TITLE
HCAP-1140 | Enable MultiOrg display of prospecting statuses

### PR DESCRIPTION
**Pull request by fwpushan**
- [x] Update status-filter logic
- [x]  Update data display logic 
- [x]  Add logic to copy site association from one prospecting status to another
- [x] Update tests
- [x] Update status-filter to support multiorg archive display 

Ticket: [HCAP-1140](https://freshworks.atlassian.net/browse/HCAP-1140)

### CODE REVIEW

**Review by taraepp — APPROVED** [review]
looks good to me
I'd like to know more about the change to the makefile. Has the change to bash fixed any bugs with make commands or what kind of impact does the change have?

**Comment by fwpushan** [comment]
> looks good to me I'd like to know more about the change to the makefile. Has the change to bash fixed any bugs with make commands or what kind of impact does the change have?
@taraepp This will allow us to shell into server container with `bash`. No, impact on code just improve usability.

**Review by dbayly-freshworks — APPROVED** [review]
Looks ok to me code wise.

File: server/services/participants-helper.js [hunk 1]
```diff
@@ -23,6 +14,10 @@ const scrubParticipantData = (raw, joinNames) =>
         ? { data: statusInfo.data }
         : {}),
       status: statusInfo.status,
+      associatedSites: statusInfo.statusSiteJoin?.map((site) => site.statusSiteDetailsJoin.body),
```
[the remark this answers is not in the document]
Comment (fwpushan) [reply server/services/participants-helper.js:17]: That was a document entity, body is the default props/column

**Pull request merged**